azext_iot/tests/iothub/test_iot_messaging_int.py

```diff
@@ -426,14 +514,15 @@ def test_hub_monitor_events(self):
                     "$.mid=12345;key0=value0;key1=1",
                     1,
                     LIVE_RG,
-                    None,
-                    0,
+                    None
                 ],
                 max_runs=1,
+                return_handle=True,
             )
+
         # Monitor events for all devices and include sys, anno, app
         self.command_execute_assert(
-            "iot hub monitor-events -n {} -g {} --cg {} --et {} -t 5 -y -p sys anno app".format(
+            "iot hub monitor-events -n {} -g {} --cg {} --et {} -t 7 -y -p sys anno app".format(
```

Comment:
No need to change or fix, just curious if the results of this test are consistently working with a timeout of `7` but not `5`.  Still better than `30` :)